tests/bench/coremark/coremark_main.c

```diff
@@ -235,6 +242,19 @@ for (i = 0; i < MULTITHREAD; i++)
 #else
 #error "Please define a way to initialize a memory block."
 #endif
+
+    ee_printf("Starting coremark %d %d 0x%x %d",
+				results[0].seed1,
+				results[0].seed2,
+				results[0].seed3,
+				results[0].iterations,
+				results[0].execs);
+	ee_printf(" data-size %d", results[0].size);
+#ifdef CONFIG_COREMARK_RUN_CNT
+	ee_printf(" run-count %d", CONFIG_COREMARK_RUN_CNT);
+#endif
+	ee_printf("\n");
+
```

Comment:
Why is this a necessary code block moving?

Reply:
fixed